modules.d/99base/module-setup.sh

```diff
@@ -19,6 +19,9 @@ install() {
     ln -s dracut-util "${initdir}/usr/bin/dracut-getarg"
     ln -s dracut-util "${initdir}/usr/bin/dracut-getargs"
 
+    # fallback when shell-interpreter is not included
+    [ ! -e "${initdir}/bin/sh" ] && inst_simple "${initdir}/bin/sh" "/bin/sh"
```

Comment:
Shouldn't that live in shell-interpreter?

Reply:
Thanks. That's a valid reason. Can we have this explanation in the commit message? Then I will approve this merge request.